<launch>
    <node pkg="xiaohu_robot" type="tts_node" name="tts_node" output="screen" />
    <node pkg="xiaohu_robot" type="asr_node" name="asr_node" output="screen" />
    <node pkg="sound_play" type="soundplay_node.py" name="soundplay_node" />
</launch>